### PR TITLE
Adds Sushi Mat and Sushi Makiroll

### PR DESCRIPTION
**Pull request by Mitchs98**
**What does this PR do:**
Makes sushi 100% more efficient and viable to make a restaurant out of. Currently, it's a pain in the ass to get any kind of sushi restaurant going. Making sushi 1 by 1 by hand is slow, and most people don't even know you can _grill_ it of all things to make more sushi faster once the grill is upgraded. And well...grilling sushi just sounds entirely unrealistic to me. It's not supposed to be grilled people!

Anyways. This PR adds Makirolls of their respective sushi type, a large sliceable variant with 4x the nutriment/protein(roll dependent) as their singular counterparts. They use one bowl of rice plus four times whatever other ingredients they need(or in the case of Tobiko Egg rolls, 4 sushi and 4 eggs. Though I am considering lowering it to two eggs...as 4 eggs is a whole lotta eggs).

To craft the Makirolls you use the hand crafting table alongside the new Sushi mat. 3 start in the chefs vendor and more can easily be crafted with 5 wood and 5 coil. I will potentially later down the line make a special Sushi cargo crate...but as of now craft the mat or ask the chef for one.

e932ae462cf193dc30d6ec8178ef9681
9cf86e7d4d967916553b9347cb9a56af
1962fb19b00230b9e209bc579e9efcae

The only sushi roll I wasn't able to make craftable was the Unagi roll, which in and of itself isn't even on the crafting menu on the first place and does require to be grilled.

**Images of sprite/map changes (IF APPLICABLE):**
9fa8dcf5ff823fb7c2a3417cd435c7f5

Credit of the Makiroll sprite goes to @Ionward 

**Changelog:**
:cl: 
add: Add's Makiroll's! Larger sliceable variants of sushi that yield four rolls of sushi in the same time that it previously took for one. You can also eat the Makiroll itself for 4x the nutriment/protein of a regular roll of sushi. Requires the Sushi Mat to craft.
add: Add's the Sushi Mat. 3 start in the chef's vendor. New ones craftable with 5 wood and 5 cable coil.
/:cl:

### CODE REVIEW

**Comment by Ty-Omaha** [comment]
your CL is broken

File: code/game/objects/items/weapons/kitchen.dm [hunk 1]
```diff
@@ -249,3 +250,17 @@
...
+/obj/item/kitchen/sushimat
+	name = "Sushi Mat"
+	desc = "A wooden mat used for efficient sushi crafting."
+	icon_state = "sushi_mat"
+	force = 5
+	throwforce = 5
+	throw_speed = 3
+	throw_range = 3
+	w_class = WEIGHT_CLASS_SMALL
+	attack_verb = list("rolled", "cracked", "battered", "thrashed",)
```
Comment (Ty-Omaha) [inline code/game/objects/items/weapons/kitchen.dm]: extra comma at the end

Comment (Mitchs98) [reply code/game/objects/items/weapons/kitchen.dm]: haha whoops. I was originally going to add more attack verbs to it but I ran out of witty ideas.

**Comment by Fox-McCloud** [comment]
Don't bother making a mat just to make sushi. It's not really necessary; just add the recipes as handcrafting and be done with it.

**Comment by Mitchs98** [comment]
> Don't bother making a mat just to make sushi. It's not really necessary; just add the recipes as handcrafting and be done with it.

It's there for RP factor and for authenticity. Sure it's not necessary, but it looks neat. Lots of things in game aren't necessary but they're there.

**Comment by Aurorablade** [comment]
Sometimes adding something for the sake of 'its neat' isn't always needed. I am going to concur with fox.

**Comment by Mitchs98** [comment]
Right. Well. Other than Aurora and Fox, who prefers a mat and who prefers just adding the makirolls? Ultimately, I just want a way to make sushi a viable gimmick. The mat was to support that, to make an authentic sushi restaurant not just because 'it's neat' but because people, sometimes, actually RP. It might not mechanically be needed but it adds to the RP factor. The bar isn't mechanically needed, but it adds to the RP factor. Boxing ring? Holodeck? All not needed, both add to RP and give crew something to do.

**Comment by Denthamos** [comment]
Honestly I would prefer the mat for RP flavor because traditional crafting places stuff at your feet, on the floor.  With the mat the items are made on the table. 

Add in it adds to aesthetic quality level when you RP as a sushi chef.  Sounds like some people just are forgetting about the RP side of things .

And secondly, if what Fox and Aurora is saying is the case, why have a microwave, grill, oven, and mixer then?  Might as well just have a single one thing that does all if this is going to be the way things are.

**Comment by Mitchs98** [comment]
> Honestly I would prefer the mat for RP flavor because traditional crafting places stuff at your feet, on the floor. With the mat the items are made on the table.
> 
> Add in it adds to aesthetic quality level when you RP as a sushi chef. Sounds like some people just are forgetting about the RP side of things .
> 
> And secondly, if what Fox and Aurora is saying is the case, why have a microwave, grill, oven, and mixer then? Might as well just have a single one thing that does all if this is going to be the way things are.

It actually doesn't place it on the table...I really wish it did! But I couldn't figure out how to get it to do that without heavily altering hand crafting code x.x I just put the roll on the table to show it off by the mat :v

**Comment by Pennwick** [comment]
This all looks good for all sushi chefs. Though I'd reduce the number of sheets of wood and coils needed to create a mat. 5 sheets means you need more than one and a half tables worth for something that isn't much larger than a sheet of paper and you're using 5 meters of cable. So unless SS13 really puts the JUMBO in jumbo sushi probably 1 plank and 2 coils would be more appropriate.

**Pull request merged**